src/domain/layout.ts

```diff
@@ -8,6 +8,7 @@ export enum PlacementKind {
   AnotherNamespace = 'AnotherNamespace',
   InsideWithConnections = 'InsideWithConnections',
   InsideWithoutConnections = 'InsideWithoutConnections',
+  OutsideManaged = 'OutsideManaged',
```

Comment:
What is it?

Reply:
Just started with this new placement type to place service cards on the right side of the map